commands/config.go

```diff
@@ -333,6 +335,10 @@ func configGetCmdF(c client.Client, _ *cobra.Command, args []string) error {
 		return errors.New("invalid key")
 	}
 
+	if cloudRestricted(config, path) && reflect.ValueOf(val).IsNil() {
+		return errors.New("configuration is restricted to the cloud")
```

Comment:
The wording is slightly incorrect. Suggestion: "changing this config path: %s is restricted in a cloud environment" and include the path as well.

Reply:
Makes sense 👍